cardano-db-sync/src/Cardano/DbSync/Era/Shelley/Validate.hs

```diff
@@ -14,11 +14,12 @@ import           Cardano.BM.Trace (Trace, logError, logInfo, logWarning)
 import           Cardano.Db (DbLovelace, RewardSource)
 import qualified Cardano.Db as Db
 import           Cardano.DbSync.Era.Shelley.ValidateWithdrawal (validateRewardWithdrawals)
+import           Cardano.DbSync.Era.Shelley.Generic.StakeCred
```

Comment:
This is imported twice in this module, both as `Generic.StakeCred` and plain `StakeCred`
the style of `Cardano.DbSync.Era.Shelley.Generic (StakeCred(..))` etc seems more common in this codebase.